org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/compiler/regression/GenericTypeTest.java

```diff
@@ -43243,6 +43243,11 @@ public void test1222() {
 		"	public class X<T extends Zork & Runnable> {\n" +
 		"	                         ^^^^\n" +
 		"Zork cannot be resolved to a type\n" +
+		"----------\n" +
+		"2. ERROR in X.java (at line 4)\n" +
+		"	Runnable r = x2.get();\n" +
+		"	                ^^^\n" +
+		"The method get() from the type X<T> refers to the missing type Zork\n" +
```

Comment:
Technically, even with the missing type, it should be discernible that the `x2.get()` is assignable to `Runnable` - no ?

Not really asking for fix here - the message is not incorrect.

Reply:
Yea, another example (after `new A().foo(new B())` above) where more detailed analysis could conclude that even when hitting a missing class we could still pull out sufficient type information for a safe decision.

I'll leave that as an exercise for the reader :)